templates/rag-redis/README.md

```diff
@@ -42,25 +46,26 @@ pip install -U langchain-cli
 To create a new LangChain project and install this as the only package, you can do:
 
 ```shell
-langchain app new my-app --package rag-redis
+langchain app new my-app
+cd my-app
 ```
 
-If you want to add this to an existing project, you can just run:
+Add this template to your project, select `y` when prompted:
```

Comment:
Was there an issue with the previous wording? Philosophy of the CLI is that people shouldn't be pushed one way or the other in case they manage dependencies another way (e.g. poetry instead of pip). If it led to confusion, would love to chat!

Reply:
Good call -- I'll revert! This was just my habit of working with them :)